src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java

```diff
@@ -67,9 +68,13 @@
     protected Settings settings;
 
     // Current maven project
-    @Parameter(defaultValue= "${session}", readonly = true)
+    @Parameter(property= "session")
```

Comment:
Not an important change.  Aligned with latest recommended practice. 

Chas

> On Jun 26, 2016, at 6:29 PM, Roland Huß <notifications@github.com> wrote:
> 
> In src/main/java/io/fabric8/maven/docker/AbstractDockerMojo.java:
> 
> > @@ -67,9 +68,13 @@
> >      protected Settings settings;
> >  
> >      // Current maven project
> > -    @Parameter(defaultValue= "${session}", readonly = true)
> > +    @Parameter(property= "session")
> Why is this change important ?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>